packages/breadboard/tests/index.node.test.ts

```diff
@@ -0,0 +1,217 @@
+// export default Ajv;
+import { Schema, SchemaDraft, ValidationResult, Validator } from "@cfworker/json-schema";
```

Comment:
Curious why you picked this validator? (No judgment, just curious -- I like that this one is ES modules!)

Reply:
AJV didn't have compatible exports. cfworker is what I settled on in my own projects after going through pretty much every alternative I could find.